source/technical/networks/networks_stitching.rst

```diff
@@ -33,73 +32,7 @@ in the ``resource_properties`` attribute. An example is provided below:
 
 .. code-block:: bash
 
-   blazar lease-create --reservation resource_type=network,network_name=my-stitchable-network,resource_properties='["==","$physical_network","exogeni"]' --start-date "2015-06-17 16:00" --end-date "2015-06-17 18:00" my-stitchable-network-lease
-
-
-Connecting Chameleon to ExoGENI
```

Comment:
It feels like this maybe needs some sort of replacement? We don't explain how to connect, e.g., TACC to UC now, which is the primary use-case. Even if the process is "mail help desk", I think it would be helpful to document

Reply:
ah, it got butchered by the diff. I pasted in your proposed syntax in the second commit, and help-desk is linked at the end, as before.